zerver/migrations/0541_merge_20240623_1158.py

```diff
@@ -0,0 +1,12 @@
+# Generated by Django 5.0.6 on 2024-06-23 11:58
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0537_merge_20240620_1545"),
+        ("zerver", "0540_remove_realm_create_private_stream_policy"),
+    ]
+
+    operations = []
```

Comment:
These database migrations are a mess. You should probably delete them all and regenerate them after rebasing; see also https://zulip.readthedocs.io/en/latest/subsystems/schema-migrations.html.